staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -725,6 +725,8 @@ message Capability {
     repeated string add_capabilities = 1;
```

Comment:
Let's explicitly say: `List of capabilities to add to the permitted, inheritable effective and bounding sets.`

Reply:
Done.